README.md

```diff
@@ -1 +1,29 @@
-# hw01
\ No newline at end of file
+# hw01
+
+Hello! My name is Frania Mendoza Lua, I am a second year **PHD student** at the *School of Social Administration*. 
+
+I study:
+
+1. immigration
+
+2. teens
+
+3. teen health behaviors
+
+# What am I doing across the Midway?
+
+I am taking this course because I am tinkering with an idea of incorporating tech into the work that I do. 
+
+# Why I study teens?
+
+I study teens because it is such an important developmental period that shapes the trajectory of our lives. For example, check out this [embarassing video](https://www.youtube.com/watch?v=OBZ_jYmedBI) of me doing my first interview on TV as a teen, it haunts me til this day.  
+
+# The Weather
+
+## OMG Chicago weather
+
+As a Southern Californian, growing up 15 minutes (30 with traffic) from the beach, living in Chicago is brutal. Naively, I chose a place 3 blocks from the lake thinking, I made it, I finally live by the water. It's a sham! I didn't know what a lake effect was until now. Living by the lake is nice for like two months, the rest of it is a mixture of sadness, coldness, and the ever ending temptation of wanting to take a walk on the lakefront but realizing its March. I low key get offended when Chicagoans call the lake a beach, and don't you dare call it a coast. 
```

Comment:
My brother-in-law moved to LA (from Northern Virginia) a few years ago. It did not take him long to acclimate. Now if it's in the 60s he has to wear a scarf and jacket to stay warm

![](https://media.giphy.com/media/NrwULs858gcms/giphy.gif)

Reply:
![](https://media.giphy.com/media/3ohs7XBjSEih0jXUv6/giphy.gif)